### DISCUSSION TITLE
Please add portal frame texture option

### DISCUSSION THREAD

**Discussion by MrChimcken** [iPortalTeam/ImmersivePortalsMod#1038] [Ideas]
Is it possible you could add a config option to turn on the default portal texture? Sense shaders can make a weird line in the middle of the portals it would be nice to have the option to mask that with (A very transparent) portal frame texture.

**Comment by qouteall** [comment]
For technical reasons, the portal overlay feature is incompatible with iris shaders

**Reply by MrChimcken** [reply]
Okay, thanks for your time.
